code/drasil-code-base/Language/Drasil/Code/Expr/Render.hs

```diff
@@ -0,0 +1,116 @@
+module Language.Drasil.Code.Expr.Render (
+    renderExpr, renderRealInterval, renderConstraint
+) where
+
+import qualified Language.Drasil as L
+import qualified Language.Drasil.Development as LD
+
+import Data.Bifunctor (Bifunctor(bimap, second))
+
+import Language.Drasil.Code.Expr
+
+-- | Render an algebraic expression into our code expression language
+renderExpr :: L.Expr -> CodeExpr
```

Comment:
`render` made sense when it was more of a printing operation, and this propagated.  A simple `toCode` might be better, but I'm open to suggestions that are less printing-oriented.

Reply:
Also, because this whole file is about 'rendering', there is no point embedding that in the name too.  So `expr`, etc, might end up being best?

Reply:
Yep, I agree. I changed the name of the function to just 'expr', and I also renamed the file to Convert.hs, I think it's a bit more suitable.